testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -202,6 +204,66 @@ public void testCreateSchema()
         assertUpdate("DROP SCHEMA " + schemaName);
     }
 
+    @Test
+    public void testRenameTable()
+    {
+        if (!hasBehavior(SUPPORTS_RENAME_TABLE)) {
+            // must reference an existing table in ALTER TABLE since name is validated before calling renameTable
+            assertQueryFails("ALTER TABLE nation RENAME TO yyyy", "This connector does not support renaming tables");
+            return;
+        }
+
+        if (!hasBehavior(SUPPORTS_CREATE_TABLE)) {
+            throw new AssertionError("Cannot test ALTER TABLE RENAME without CREATE TABLE, the test needs to be implemented in a connector-specific way");
+        }
+
+        String oldTable = "test_rename_" + randomTableSuffix();
+        String newTable = "test_rename_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + oldTable + " (a bigint, b double)");
+        assertUpdate("ALTER TABLE " + oldTable + " RENAME TO " + newTable);
+        assertThat(query("SHOW TABLES LIKE '" + oldTable + "'")).returnsEmptyResult();
```

Comment:
please be consistent:
either `.returnsEmptyResult();` from new line or on the same line everywhere.

Reply:
I copied this from the existing `renameTable` test and hadn't noticed. Fixed in both tests.